episodes/03-audience.md

```diff
@@ -22,48 +19,133 @@ adapted to place an emphasis on assessing learning during a workshop:
-
-[The Carpentries model of how people gain expertise][it-skill-acquisition]: novice -> competent practitioner -> expert
-
-- Why this matters when designing a new lesson
-  - Prior knowledge of audience mental models: what do they already know?
-  - Novices and competent practitioners need to be taught differently
-
-Your lesson will also be more effective if it aligns with the motivations of the target audience
-(quote from/link to Ambrose et al).
-
-Think about who your lesson is for. Be specific.
-
-- What kind of background might your learners have?
-- What problems are they facing that your lesson can help them solve?
-- What do they already know about the topic?
-- What vocabulary do they use? I may be that the terms you will teach have different meaing in the learner's research domain.
-- What other considerations might you need to make e.g. their primary language differ from yours, etc?
-
-A callout about mastery rubrics?
+Thinking carefully about the target audience will help with this.
+Prominently displaying a description of the target audience
+will also help attract people with the right motivation and 
+relevant prior knowledge to attend your workshops.
+
+### Expertise
+
+One of the most important things we can identify about our target audience
+is the level of expertise they will already have in relation to the skills taught by your lesson.
+In The Carpentries Instructor Training curriculum,
+we describe 
+[three different stages of skill acquisition: novice, competent practitioner, and expert][it-skill-acquisition].
+
+Briefly, the novice is someone who does not know what they do not know, i.e., they do not yet know what the key ideas in the domain are or how they relate, the competent practitioner has enough understanding for everyday purposes, and the expert is someone who can easily handle situations that are out of the ordinary.
+
+When designing a new lesson, it is important to think about 
+the level of expertise that you expect learners to arrive with for two reasons:
+
+1. It helps to predict what prior knowledge and 
+   [mental model][it-mental-models] learners will have 
+   of the lesson domain when they arrive. 
+   This can enable you to make progress quickly
+   by 
+   a) working to help learners recall (_activate_) that prior knowledge,
+   b) building on the conceptual understanding they already have and, 
+   c) perhaps most importantly,
+   giving you some idea of what _misconceptions_ they might arrive with.
+   It is vital that misconceptions are identified and corrected early on,
+   before learners try to incorporate new knowledge into a broken mental model.
+2. People at different stages of this process need to be taught differently.
+   For example, novices will learn more from lessons that include worked examples,
+   but the same approach may actually hinder learning for competent practitioners.
```

Comment:
```suggestion
2. People at different stages of this process need to be taught differently.
   For example, novices will learn more from lessons that include worked examples and are more tutorial-like with step by step explanations of the process and focussed on a specific task but without too many facts, but the same approach may actually hinder learning for competent practitioners who may prefer factual manuals that are well suited for troubleshooting and intended to give specific information.
```

Reply:
Always liked this tutorial/manual analogy :-).

Reply:
What I am trying to get at here is the [_expertise reversal effect_](https://en.wikipedia.org/wiki/Expertise_reversal_effect). How about this?

```suggestion
2. People at different stages of this process need to be taught differently.
   For example, novices will learn more from lessons that include worked examples 
   and are more tutorial-like i.e. focused on a specific task, 
   with step-by-step explanations of the process,
   but without a lot of extra information that is not directly relevant. 
   However, the same approach may actually hinder learning for competent practitioners 
   who may be distracted by a step-by-step explanation 
   of something they already have the prior knowledge to understand on their own.
   For learners at this level of expertise, 
   lessons which include activities offering learners 
   the freedom to explore options and develop their own solutions,
   are likely to be more effective.
```

Reply:
Sounds like a good re-write to me. 👍